ipatests/test_integration/test_krbtpolicy.py

```diff
@@ -177,3 +185,32 @@ def test_krbtpolicy_otp(self):
             reset_to_default_policy(master, USER1)
             self.master.run_command(['rm', '-f', armor])
             master.run_command(['ipa', 'config-mod', '--user-auth-type='])
+
+    def test_krbtpolicy_jitter(self):
+        """Test jitter lifetime with no auth indicators"""
+        tasks.kinit_admin(self.master)
+        kinit_check_life(self.master, USER1)
+        tasks.kdestroy_all(self.master)
+
+    def test_krbtpolicy_jitter_otp(self):
+        """Test jitter lifetime with OTP"""
+        tasks.kinit_admin(self.master)
+        self.master.run_command(["ipa", "user-mod", USER1,
+                                 "--user-auth-type", "otp"])
+        self.master.run_command(["ipa", "config-mod",
+                                 "--user-auth-type", "otp"])
+        kinit_check_life(self.master, USER1)
+        self.master.run_command(["ipa", "config-mod", "--user-auth-type="])
```

Comment:
You can move resetting the global auth type back to empty using a pytest fixture:

```
@pytest.fixture
def reset_policy(self):
    yield
    self.master.run_command(["ipa", "config-mod", "--user-auth-type="])
    tasks.kdestroy_all(self.master)
```

And define each test ala:
```
def test_krb5policy_foo(self, reset_policy)
```

Reply:
Done, thanks for the suggestion!